src/Microsoft.Framework.Runtime/PatternsCollectionHelper.cs

```diff
@@ -53,7 +58,12 @@ private static IEnumerable<string> CreateCollection(string projectDirectory, par
             {
                 if (Path.IsPathRooted(pattern))
                 {
-                    throw new InvalidOperationException(string.Format("Patten {0} is a rooted path, which is not supported.", pattern));
+                    throw new InvalidOperationException($"Pattern {pattern} is a rooted path, which is not supported.");
+                }
+
+                if (literalPath && pattern.Contains('*'))
+                {
+                    throw new InvalidOperationException($"Pattern {pattern} is not expected to contain wildcard characters.");
```

Comment:
It's not needed.

The exception comes with line and column information. When error is shown in the VS the error list will direct you to the exactly line of code. In addition, the line and column are also shown when application is running from command line.

Given that, I think the property name is important but not significant enough to change the signature.